app.py

```diff
@@ -13,18 +16,24 @@
 from view.main_content_panel import MainContent
 from view.status_panel import Status
 
-# The log file is stored inside the app directory (on Mac, it is inside the package)
-logging_file_path = Path(os.path.join(os.path.dirname(__file__), "rs_log.log"))
+# The log file is stored inside the RimSort install directory (on Mac, it is inside the package)
+data_path = os.path.join(os.path.dirname(__file__), "data")
+logging_config_path = os.path.join(data_path, "logging_config.json")
+logging_file_path = os.path.join(data_path, "RimSort.log")
 
-# Only the most recent log file is kept
-logging_file_path.unlink(missing_ok=True)
+if platform.system() == "Linux":
```

Comment:
Why are we switching logging implementations? Have you verified that we aren't losing any information here? Can you add comments describing the purpose of this change?

Reply:
This does not change logging implementations. This is still using `logging` module, but instead of initializing the logger through the actual `logger` module, we are doing so through another module called `logger_tt`

https://github.com/Dragon2fly/logger_tt

Yes I have verified that we are not losing information - feel free to compare 2 logs yourself

As for the "comments describing the purpose of this change":

![image](https://user-images.githubusercontent.com/2766946/226515191-ccc2772c-6e38-44ae-b42d-57d250be50a4.png)

There is also discussion in the Discord